Provider/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -0,0 +1,157 @@
+package com.spotify.confidence
```

Comment:
Shall we consider splitting the Confidence stuff into a separate module early? Would doing this later cause friction for our users, when updating our library?